spec/features/users/guest_can_create_account_spec.rb

```diff
@@ -40,7 +49,7 @@
       visit new_user_path
 
       expect(current_path).to eq(user_path(user))
-      expect(page).to have_content("You are already logged in.  Logout first to create a new account.")
+      # expect(page).to have_content("You are already logged in.  Logout first to create a new account.")
```

Comment:
Looks like the alert messages are being rendered by the Materalize/Toast but wanted to make sure that was the case, and if so can we delete the commented out "expect"s?